show/main.py

```diff
@@ -1356,16 +1356,29 @@ def show_run_snmp(db, ctx):
 @runningconfiguration.command()
 @click.option('--verbose', is_flag=True, help="Enable verbose output")
 def syslog(verbose):
-    """Show Syslog running configuration"""
+    """Show Syslog running configuration
+    To match below cases:
+    *.* @IPv4:port
+    *.* @[IPv4]:port
+    *.* @[IPv6]:port
```

Comment:
>port [](http://example.com/codeflow?start=16&length=4)

The port is optional and the default is 514. ref: [https://linux.die.net/man/5/rsyslog.conf](https://linux.die.net/man/5/rsyslog.conf)

Reply:
I can make the port optional.